README.md

```diff
@@ -1,27 +1,55 @@
-# privacy-evaluator
+# Privacy Evaluator
+<!--[![Gitter](https://badges.gitter.im/fairlearn/community.svg)](https://gitter.im/fairlearn/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)-->
+
+- space for the description of our project (core idea)
+
+- For example:
+The privML Privacy Evaluator is a tool that assesses ML model's levels of privacy by running different attacks on it. .....
+
+## Dependencies
```

Comment:
Please do not remove the things that belong here. The license, code style, python version, and contact over gitter definitely need to be in the readme file!

Reply:
The relevant version of the README can be found in the pull request "Update of README.md #32". This pull request is only concerning PIA interface and not up to date on the README.